File: 10872.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        int seedNum = sc.nextInt();
        System.out.println(factorial(seedNum));
    }
    public static int factorial(int inputNum){
        int result = 1;
        for(int i = inputNum; i > 0; i--){
            result = result * i;
        }
        return result;
    }
}

File: 10872.c
#include<stdio.h>

int factorial(int inputNum);

int main(){
    int seedNum = 0;
    scanf("%d", &seedNum);
    printf("%d", factorial(seedNum));
    return 0;
}

int factorial(int inputNum){
    int result = 1;
    for(int i = inputNum; i > 0; i--){
        result = result * i;
    }
    return result;
}
